cmd/main.go

```diff
@@ -130,9 +130,11 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			slog.String("error", err.Error()),
 			slog.String("attempting to marshal", fmt.Sprintf("%+v\n", response)))
 		errorResponse, _ := json.Marshal(newErrorReport(err.Error()))
+		w.Header().Set("Content-Type", "application/json")
 		w.Write([]byte(errorResponse))
 	} else {
 		slog.Debug("Responding to request", "response", responseJson)
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
`w.Header().Set("Content-Type", "application/json")`

could be called just once, before checking for errors, as the response is always a JSON.

Reply:
Updated